packages/firestore/test/unit/local/local_store.test.ts

```diff
@@ -1997,6 +2005,22 @@ function genericLocalStoreTests(
       .finish();
   });
 
+  it('can handle multiple field patches on local docs', () => {
```

Comment:
this isn't handling multiple patches. It's just one patch isn't it?

The name used in Android is `testUpdateOnRemoteDocLeadsToUpdateOverlay`. I think a better name here would be `update on remote doc leads to update overlay`